ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/replacehook.js

```diff
@@ -48,6 +48,7 @@
         irreplaceable = ['file-input'],
         editableJsonPath = '.model.json',
         componentJsonPath = '.json';
+    const dialogCssClass = 'cmp-replace-dialog-search-components';
```

Comment:
Since this is moved to cq-guides, can you close this PR please ?

Reply:
@egmox I hope these changes are already present in cq-guides.

Reply:
yes, they are present in cq-guides and merged